snap/snapcraft.yaml

```diff
@@ -66,8 +66,8 @@ description: |
   **Cryptography**
   ---
 
-  The snap installs the CUDA keyring, verifying its integrity using `curl` and SHA256 checksums.
-  The keyring also sets up the appropriate source for the DCGM deb package.\
+  The snap installs the CUDA keyring deb package using `curl` and verifies its integrity using SHA256 checksums.
```

Comment:
nit: we install via dpkg, not curl

Reply:
But the keyring is installed using `curl`

Reply:
We fetch a debian pkg named `cuda-keyring_1.1-1_all.deb`, check the SHA256SUM and after installing the deb package that the sources are going to change